### ISSUE TITLE
Add hotkeys

### ISSUE THREAD

**Issue by kkrysiak** [griffithlab/igvnav#3]
Hotkeys would be great for saving, next variant, sort and calls (S/G/F/A) to provide less clicking. Not sure it makes sense for all of the tags but perhaps others could provide feedback. @LMT337 @zlskidmore @kcampbel

**Comment by kcampbel** [comment]
I agree! Could we set it so that the arrow keys move to the next variant?

**Comment by ahwagner** [comment]
Great idea guys. I think a nice way to do this is have hotkeys user-customizable, with defaults as suggested in this thread.

**Comment by fgomez02** [comment]
This sounds like a really good idea to me.  I love the idea of less
clicking!
